### ISSUE TITLE
pytest conflict when having another test folder in the directory

### ISSUE THREAD

**Issue by pschicho** [Wall-Go/WallGo#325]
When storing also `WallGoCollision` inside the WallGo main folder i.e.

```
-Wallgo
  - tests
  - WallGoCollision
```

there is a mismatch between `conftest.py` files when calling `pytest -v`. Since I am not sure if this has to do with my own setup or the pytest one, I wanted to list this as a potential issue. It results in an error that is only resolved when removing the sub test folder in `WallGoCollisions`.

**Comment by niemilau** [comment]
Hmm, I don't think this should even work in the first place because WallGoCollision has its own pytest.ini. Why do you have WallGoCollision inside the WallGo folder? They are separate repos on purpose now, although on hindsight some submodule setup might've been better.

**Comment by og113** [comment]
I don't think this is an issue with WallGo, but rather with your specific setup. `pytest` looks for files and folders starting with `test` or `tests`. I would guess that your setup here breaks the rules somehow for where `pytest` looks for tests.
